sway-fmt-v2/src/config/comments.rs

```diff
@@ -1,7 +1,6 @@
 //! Configuration options related to formatting comments.
-use crate::constants::DEFAULT_MAX_COMMENT_WIDTH;
-
 use super::user_opts::CommentsOptions;
+use crate::constants::DEFAULT_MAX_COMMENT_WIDTH;
```

Comment:
Do we want to keep this kind of separate imports rooted from `super::`? If we also convert those imports to `crates::` type (and add it to the others) they would also be ordered by `rustfmt`. But I would also understand if you prefer having these separated as it is a shorter relative path. 

```suggestion
use crate::{config::user_opts::CommentsOptions, constants::DEFAULT_MAX_COMMENT_WIDTH};
```

Reply:
Personally I like `super` (: if @mitchmindtree or @JoshuaBatty prefer `crate` only then I'd gladly change it. But I have no qualms as is!

Reply:
I think `crate` is perhaps a little more conventional, though `super` can be useful when you're in a module that you know will always be nested and have the same relative path, e.g.

```rust
// Inside module `foo.rs`

fn foo() { ... }

mod tests {
    use super::foo;
    // ...
}
```

^ Even if `foo.rs` gets moved or nested within another module, `tests` will always be nested within `foo.rs` so it's pretty safe to use `super` from within `tests`.

For most of the cases in this lib I'd personally go with `crate`.

Reply:
Yeah I prefer `crate`, makes it a bit more flexible for the reasons mitchie stated above.

Reply:
gotcha, I'll add those changes in the morning